options/locale/locale_nl-NL.ini

```diff
@@ -297,13 +297,13 @@ email_deletion_desc=Dit e-mailadres verwijdert, worden gerelateerde informatie v
 email_deletion_success=E-mail is succesvol verwijderd!
 add_new_email=Nieuw e-mailadres toevoegen
 add_email=E-mailadres toevoegen
-add_email_confirmation_sent=Een nieuwe bevestiging e-mail werd verstuurd naar '%s', gelieve uw inbox in de komende %d uren te controleren om het bevestigingsproces te voltooien.
+add_email_confirmation_sent=Een nieuwe bevestigings e-mail werd verstuurd naar '%s', gelieve uw inbox in de komende %d uren te controleren om het bevestigingsproces te voltooien.
 add_email_success=Het e-mailadres was toegevoegd.
 
 manage_ssh_keys=Beheer SSH sleutels
 add_key=Sleutel toevoegen
 ssh_desc=Dit is een lijst van alle SSH sleutels die gekoppeld zijn aan uw account. Verwijder alle sleutels die u niet herkent.
-ssh_helper=<strong>De behoeftehulp van?</strong> Check out onze gids voor het <a href="%s"> genereren van SSH sleutels</a> of <a href="%s"> algemene SSH</a> problemen.
+ssh_helper=<strong>Weet je niet hoe?</strong> Lees dan onze handleiding voor het <a href="%s"> genereren van SSH sleutels</a> of voor <a href="%s"> algemene SSH</a> problemen.
```

Comment:
ssh_helper=<strong>Weet u niet hoe?</strong> Lees dan onze handleiding voor het <a href="%s"> genereren van SSH sleutels</a> of voor <a href="%s"> algemene SSH</a> problemen.